src/buildApplication/fixPrerenderedRoutes.ts

```diff
@@ -83,6 +83,7 @@ async function getRoutePath(
  * // index.prerender-fallback.html -> index.html
  * // index.rsc.prerender-fallback.rsc -> index.rsc
  * // favicon.ico.prerender-fallback.body -> favicon.ico
+ * // 1.json.prerender-fallback.json -> 1.json
```

Comment:
why `1`? looks a bit random :sweat_smile: (maybe `X` of the name of some json file we always get?)

Reply:
that was the file i was basing it off of :p

Reply:
ah ok :sweat_smile::+1: